playground/routes/config.ts

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line require-await
+export default eventHandler(async (event) => {
```

Comment:
Please avoid commiting playground changes when making a PR. (you can revert them now)